float64 uv_index
float64 variance